src/en/mobile/android/development/vocal-navigation.md

```diff
@@ -0,0 +1,463 @@
+    myView.announceForAccessibility(getString(R.string.criteria_contentchange_ex1_announce))
+    }
+</code></pre>
+
+**Example of a live region**:
+
+<pre><code class="xml">&lt;TextView
+   […]
+   android:accessibilityLiveRegion="polite" /&gt;
+</code></pre>
+
+
+## Do not vocalize the decorative and hidden elements.
+
+**Target&nbsp;:** everyone and in particular the visually impaired.  
+**When&nbsp;:** from design,  and during development.
+
+**Description&nbsp;:**  
+
+To not confuse navigation with a screen reader, it is necessary to vocalize only the significant elements. Decorative elements, such as illustrations, should not be vocalized.
+
+Although invisible on the screen, some elements could be read by the screen reader (elements positioned outside the visible area or hidden by other elements). Overlaying frames is common, but it creates very heavy accessibility problems if it is not done correctly.
+
+A frequent case of this problem: fragments.  A bad use of fragments can lead to an impossible navigation with the screen reader.  
+  
+See the article on [use of fragments](https://developer.android.com/guide/components/fragments.html) for more information.
+
+**To be verified&nbsp;:**
+- No invisible elements are vocalized
+- Decorative elements are ignored and are not vocalized.
+
+**Examples&nbsp;:**
+
+<img src="../../../images/image_ex.png" alt="complete example of decorative image and informative icon" width="400" height="149">
+
+By decomposing the image&nbsp;:
+- <img src="../../../images/montagnard.png" alt="example of a decorative image" width="256" height="95"> pas de `contentDescription`
+- <img src="../../../images/settings.png" alt="example of informative icon - settings" width="48" height="48"> `imageView.setContentDescription("paramètres")`
+- <img src="../../../images/edit.png" alt="xample of informative icon - edit" width="48" height="48"> `imageView.setContentDescription("éditer le nom de l’image")`
+
+<pre><code class="xml">
+    &lt;ImageView
+        android:id="@+id/montagnard"
+        android:src="@drawable/montagnard"
+        android:contentDescription="@null"
+        ... /&gt;
+    &lt;ImageButton
+        android:id="@+id/settings"
+        android:src="@drawable/settings"
+        android:contentDescription="@string/settings"
+        ... /&gt;
+    &lt;ImageButton
+        android:id="@+id/edit"
+        android:src="@drawable/edit"
+        android:contentDescription="@string/edit"
+        ... /&gt;
+</code></pre>
+
+
+**Reference <abbr>WCAG</abbr>&nbsp;:**  
+- <a lang="en" href="https://www.w3.org/TR/WCAG21/#non-text-content">1.1.1 Non-text Content</a>
+
+
+## Group the elements 
+
+**Target&nbsp;:** everyone and in particular the visually impaired.  
+**When&nbsp;:** from design,  and during development.
+
+**Description&nbsp;:**  
+
+It is recommended to group the linked elements for vocalization when this increase efficiency and comprehension.
```

Comment:
s/group the linked elements/group similar elements/
(suggestion, I'm not sure of how a native speaker would say that actually)